### PR TITLE
PXB-2731 Add information about why a server check is needed (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M  source/using_xtrabackup/comparison.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst [hunk 1]
```diff
@@ -3,7 +3,13 @@
 Server Version and Backup Version Comparison
 =============================================
 
-|PXB| 8.0.21 adds the ``--no-server-version-check`` parameter. This parameter compares the source system version to the |PXB| version. 
+**Percona XtraBackup** needs specific *MySQL* features to process a backup or a restore. Unfortunately, *MySQL* can modify these features in a release, and **XtraBackup** fails until the next release.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst]: I would remove "Unfortunately"

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst [hunk 2]
```diff
@@ -3,7 +3,13 @@
...
+
+.. seealso:: :ref:`how_xtrabackup_works`
+
+To be safe, ensure that you use a **Percona XtraBackup** version equal to or above your source system version. For example, if your **Percona Server for MySQL** version is 8.0.27, you can use **Percona XtraBackup** version 8.0.28 or lower. 
+
+**Percona XtraBackup**  8.0.21 adds the ``--no-server-version-check`` parameter to compare the source system version to the **Percona XtraBackup** version to help.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst]: I would remove "to help"

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst]: Needs a transition phrase.

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst]: Rewritten

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst [hunk 3]
```diff
@@ -3,7 +3,11 @@
 Server Version and Backup Version Comparison
 =============================================
 
-|PXB| 8.0.21 adds the ``--no-server-version-check`` parameter. This parameter compares the source system version to the |PXB| version. 
+A *MySQL* change to features, such as the structure of a redo log record, can cause older versions of **Percona XtraBackup** to fail. To ensure that you can backup and restore your data, use a **Percona XtraBackup** version that is equal to or above your source server version.
+
+.. seealso:: :ref:`how_xtrabackup_works`
+
+*Percona XtraBackup* 8.0.21 adds the ``--no-server-version-check`` option.Before the backup starts, XtraBackup compares the source system version to the *Percona XtraBackup* version. If the source system version is greater than the XtraBackup version, XtraBackup stops the backup and returns an error message. This comparison prevents a failed backup or a corrupted backup due to source system changes.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst]: option.Before - add space

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/using_xtrabackup/comparison.rst]: Fixed

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**